posthog/api/capture.py

```diff
@@ -1,6 +1,7 @@
+import datetime as dt
```

Comment:
Why not keep the `from datetime import datetime` line? Avoids using `dt.datetime` below...